File: C/lt_01_estseq/lt01_estseq_07.c
#include <stdio.h>
// Lucas Sartorelli
// Calculating Volume of a paralelepiped

int main() {

  float com, alt, lar;
  printf("Insira o Valor da Altura: ");
  scanf("%f", &alt);

  printf("Insira o Valor do Comprimento: ");
  scanf("%f", &com);

  printf("Insira o Valor da Largura: ");
  scanf("%f", &lar);

  printf("O valor do Volume será %.2f.", (alt * com * lar));

  return 0;
}

File: Java/LT01_ESTSEQ/LT01_ESTSEQ_07.java
/*
 * Nome Do Programador: Lucas Sartorelli
 * Data : 13/02/2025
 * Descrição: Receba os valores do comprimento, largura e altura de um paralelepípedo. 
 * Calcule e mostre seu volume.
 */
import javax.swing.JOptionPane;
public class LT01_ESTSEQ_07 {
	public static void main(String args[]) {
		Double comprimento;
		comprimento = Double.parseDouble(JOptionPane.showInputDialog("Insira o valor do comprimento: "));
		comprimento *= Double.parseDouble(JOptionPane.showInputDialog("Insira o valor da largura: "));
		comprimento *= Double.parseDouble(JOptionPane.showInputDialog("Insira o valor da altura: "));
		JOptionPane.showMessageDialog(null,"O valor do volume é " +comprimento);
	}
}
